corehq/apps/app_manager/views/forms.py

```diff
@@ -283,6 +283,8 @@ def should_edit(attribute):
                     pass
             if xform:
                 save_xform(app, form, xform)
+                case_references = json.loads(request.POST.get('references', "{}"))
+                _update_case_refs_from_form_builder(form, case_references)
```

Comment:
pulled from same workflow in patch_xform https://github.com/dimagi/commcare-hq/blob/6421b8675cad068322ec9edbc564948231148152/corehq/apps/app_manager/views/forms.py#L373